presto-docs/src/main/sphinx/release/release-0.238.rst

```diff
@@ -0,0 +1,32 @@
+=============
+Release 0.238
+=============
+
+General Changes
+_______________
+* Fix case sensitivity issue in SQL function parameters. Function parameters should be treated case-insensitive.
+* Add ``target_result_size`` session property to customize data batch sizes being streamed from coordinator.
+* Add optimization to push null filters to the INNER side of equijoins. The optimization can be enabled with `optimize-nulls-in-joins`.
+* Add sessionProperty override to Presto JDBC URI.
+* Add support to create external function (this does not include external function execution).
```

Comment:
Remove the content in the parenthesis. We introduce what is included in the release, not what is not included.